File: Code/SimpleBudget.API/Models/Reports/SummaryModel.cs
namespace SimpleBudget.API.Models
{
    public class SummaryModel
    {
        public List<SummaryWalletModel> Wallets { get; set; } = default!;
        public List<SummaryCurrencyModel> Currencies { get; set; } = default!;

        public string ValueFormatCAD { get; set; } = default!;
        public decimal TaxCAD { get; set; }
    }
}

File: Code/client/src/api/models/SummaryModel.ts
import { SummaryCurrencyModel } from "./SummaryCurrencyModel";
import { SummaryWalletModel } from "./SummaryWalletModel";

export interface SummaryModel {
    wallets: SummaryWalletModel[];
    currencies: SummaryCurrencyModel[];
    valueFormatCAD: string;
    taxCAD: number;
}
